### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by DutchPatrick** [ppalex7/SourceComms#66]
Can you make this for Garry's Mod?

**Comment by ppalex7** [comment]
No. I haven't Garry's Mode and free time for developing sourcecomms.--Александр Дуплищев.  19.04.2015, 23:41, "DutchPatrick" notifications@github.com:Can you make this for Garry's Mod?
